docs/ATLAS.md

```diff
@@ -0,0 +1,23 @@
+### Setup Atlas using terraform ###
+
+Follow these steps to setup a M2 cluster in Atlas to be used by the uDeploy portal.
+
+1. Register an API key in Atlas.  `Atlas -> Project -> Access management -> API Keys`
+    
+    - Click `Manage -> Create API Key`
+    - Enter a short description for this key
+    - Give the key `Project Owner` permission (This permission will be reduced at the end of these steps)
+    - Click `Next`
+    - Copy the private key and store it in a secure place (This will be the only time this key will be visible and available to be copied)
```

Comment:
"Save the private key for later use" may be all we need to mention since where they save it is up to them.